addon/components/discover-page/template.hbs

```diff
@@ -0,0 +1,180 @@
+                <div class="col-xs-10 col-xs-offset-1" style="text-align: center;">
+                    {{#if (not theme.isProvider)}}
+                        <h3>{{t 'eosf.components.discoverPage.partnerRepositories'}}</h3>
+                        <div class="row">
+                            {{provider-carousel
+                                providers=fetchedProviders
+                                lightLogo=false
+                            }}
+                        </div>
+                    {{/if}}
+                </div>
+            </div>
+        {{/if}}
+
+        <div class="row p-v-sm">
+            {{#if (not showActiveFilters)}}
+                <div class="col-xs-6">
+                    {{!CLEAR FILTERS BUTTON - ONLY IF ACTIVE FILTERS NOT SHOWN}}
+                    <button {{action 'clearFilters'}} class='btn btn-default clear-filters'>
+                        {{clearFiltersButton}}
+                    </button>
+                </div>
+            {{/if}}
+
+            <div class="col-sm-6 pull-right">
+                <div class="dropdown pull-right">
+                    {{!SORT BY BUTTON}}
+                    <button class="btn btn-default dropdown-toggle" type="button" id="sortBy" data-toggle="dropdown"
+                        aria-haspopup="true" aria-expanded="true">
+                        {{t 'eosf.components.discoverPage.sortBy'}}: {{sort-option-display sortOptions sort}}
+                        <span class="caret"></span>
+                    </button>
+                    <ul class="dropdown-menu text-right" aria-labelledby="sortBy" id="sortByOptionList">
+                        {{#each sortOptions as |sortChoice|}}
+                            <li class="listOption" style="cursor: pointer; cursor: hand"><a {{action "selectSortOption" sortChoice.sortBy}}>{{sortChoice.display}}</a></li>
+                        {{/each}}
+                    </ul>
+                </div>
+            </div>
+        </div>
+
+    </div>
+</div>
+<div class="search-main p-v-xl">
+    <div class="container">
+        <div class="row search-results-section">
+            <div class="col-sm-4 col-xs-12 search-sidebar"> {{!LEFT PANEL - SEARCH FACETS}}
+                {{! ACTIVE FILTERS - FOR PREPRINTS, REGISTRIES}}
+                {{#if showActiveFilters}}
+                    <div id="activeFilters" class="well well-sm p-md">
+                        <div class="row">
+                            <div class="col-sm-6">
+                                <h4>{{t "eosf.components.discoverPage.activeFilters.heading"}}:</h4>
+                            </div>
+                            <div class="col-sm-6">
+                                <span class="pull-right">
+                                    {{!CLEAR FILTERS BUTTON}}
+                                    <button type="button" class="btn btn-default btn-sm" {{action "clearFilters"}}>{{clearFiltersButton}} {{fa-icon "remove"}}</button>
```

Comment:
This icon is on the "Clear Filters" button, which says "Clear Filters".  The remove icon is aria-hidden=true.   I don't think the user needs the remove icon to be visible since the button itself takes care of it.  I have added other aria-labels where it makes sense for other font-awesome icons - thanks for the reminder.